client/src/Components/DashboardHeader.js

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { getDashboardAllDataQuery } from '../queries/queries';
+import LoadingSpinner from './LoadingSpinner';
 import { Statistic, Header, Divider } from 'semantic-ui-react';
 
-const DashboardHeader = () => {
-  return (
-    <div className="hero-img">
-      <Header as="h1">Welcome back TODO!</Header>
-      <div className="hero-img-car-data">
-        <h3>TODO</h3>
-        <Divider hidden />
-        <Statistic.Group size="mini">
-          <Statistic>
-            <Statistic.Value>TODO</Statistic.Value>
-            <Statistic.Label>Trips </Statistic.Label>
-          </Statistic>
-          <Statistic>
-            <Statistic.Value>TODO KM</Statistic.Value>
-            <Statistic.Label>Total Distance</Statistic.Label>
-          </Statistic>
-          <Statistic>
-            <Statistic.Value>TODO</Statistic.Value>
-            <Statistic.Label>Time in Car (hrs) </Statistic.Label>
-          </Statistic>
-        </Statistic.Group>
+export default function DashboardHeader(props) {
+  const { loading, error, data } = useQuery(getDashboardAllDataQuery, {
+    variables: {
+      vehicleId: `${props.state}`,
+    },
+  });
+  if (loading) {
+    return <LoadingSpinner />;
+  } else if (error) {
+    return <p>Error! {error}</p>;
+  } else {
+    return (
+      <div className="hero-img">
+        <Header as="h1">Welcome back!</Header>
```

Comment:
can we put the user's first name here?

Reply:
Will resolve this tomorrow, need a bit more time to do it